docs/rfc/rfc-3.adoc

```diff
@@ -0,0 +1,119 @@
+and `weight = newWeight`.
+
+At any time, an operator can withdraw their collected rewards,
+transferring `rewardAvailable` T tokens to them,
+and setting their `rewardAvailable = 0`.
+
+=== Wallet action rewards
+
+Whenever a wallet is required to perform some transaction on-chain,
+a _transaction reward_ is reserved for the wallet.
+When the transaction is correctly submitted,
+the transaction reward is divided
+between the submitter and the rest of the wallet members.
+If the transaction is submitted immediately,
+the full transaction reward is paid out
+and the non-submitter members receive a larger share.
+If the transaction is submitted later,
+the submitter gets a larger reward,
+the other members receive less,
+and some of the reward is withheld from the wallet members altogether.
+The withheld amount is either paid to the sortition pool,
+or kept for future transaction rewards.
+
+For a precise calculation,
+we can use e.g. the Random Beacon entry submission reward rules.
+
+==== Earmarking
+
+Because wallet actions are tied to specific BTC deposits or redemptions,
+it is easy to earmark a specific amount of tBTC for the actions.
+As an example,
+if depositing some amount of BTC takes 0.30% of the deposit as a fee,
+this fee could be divided as follows:
+
+- 0.05% for sweeping the deposit
+- 0.05% for redeeming the deposit (or interwallet transfers)
+- 0.17% for sortition pool members
+- 0.03% for maintenance, e.g. new wallet creation
+
+If someone were to deposit 1 BTC,
+the fee would be 0.003 BTC
+of which 0.0005 BTC would be earmarked for sweeping
+and another 0.0005 BTC for redemption.
+These fees could be collected in the form of TBTC tokens
+immediately upon sweeping the deposit transaction
+and either paid out (sweeping reward)
+or kept in the wallet for future use (redemption reward).
+The remaining 0.002 TBTC
+would be used for sortition pool and maintenance rewards.
+
+=== Maintenance rewards
+
+For global maintenance actions such as new wallet creation,
+we need somebody to submit the transactions,
+but it doesn't have to be an operator.
+In these cases we can have a _maintenance reward pool_ for each task.
+
+Whenever fees are collected by the tBTC system,
+some fraction of the fees would be added to each maintenance reward pool.
+
+When the specified task is successfully performed,
+the sender of the transaction is rewarded
+with some fraction (or all) of the maintenance reward pool for the task,
+according to the reward rules of that particular maintenance task.
+
+If a part of the maintenance reward is withheld
+the withheld amount should be paid out as a sortition pool reward.
+This is distinct from paying out a fraction;
+e.g. the maintenance reward pool might pay 20% of its contents each time,
+but in some circumstances 50% of this might be withheld,
+resulting in 10% being paid to the submitter of the transaction
+and another 10% being withheld and paid to the sortition pool instead.
```

Comment:
I think it's worth writing out the incentive behind the submitter/sortition pool split (and how that helps the market determine how often maintenance needs to be done).